Dockerfile

```diff
@@ -1,5 +1,8 @@
 FROM alpine:3.9
 
+RUN apk update && \
+    apk add ca-certificates
```

Comment:
Use this instead, which avoids caching the index and performs an update implicitly:

`RUN apk --no-cache add ca-certificates`

Reply:
I'm almost tempted to do a FROM scratch but you probably use alpine for troubleshooting so I'm not changing it.